Configurations/windows-makefile.tmpl

```diff
@@ -918,6 +918,7 @@ EOF
      my $deps = join(" ", @objs, @ress, @defs, @deps);
      my $import = platform->sharedlib_import($lib);
      my $dll =  platform->sharedlib($lib);
+     my $dlldir = dirname($dll);
```

Comment:
shall this change be part of this PR? I'm failing to see how it is related to atexit/library reference counting.

Reply:
same reasoning as above.